src/backend/utilities/encryptMessage.ts

```diff
@@ -1,36 +1,31 @@
-import {
-  IDidKeyDetails,
-  IEncryptedMessage,
-  MessageBody,
-} from '@kiltprotocol/types';
+import { DidKey, IEncryptedMessage, MessageBody } from '@kiltprotocol/types';
 import { Message } from '@kiltprotocol/messaging';
-import { DefaultResolver } from '@kiltprotocol/did';
 
 import { fullDidPromise } from './fullDid';
 import { encryptionKeystore } from './keystores';
 import { configuration } from './configuration';
 
 export async function encryptMessage(
   message: Message,
-  encryptionKeyId: IDidKeyDetails['id'],
+  encryptionKeyId: DidKey['id'],
```

Comment:
If this is the full key URI, then it's a DidPublicKey['id']. If it's just the ID, then `message.encrypt()` below must use the full URI and not just the ID. If the tests work, I assume the type annotation here is not correct.